scripts/l10n-fetch-lint-compile.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# Run this from the project root--not from this directory!
+
+git clone https://github.com/mozilla-l10n/sumo-l10n.git locale
+cd locale
+
+postatus_file=../kitsune/sumo/static/postatus.txt
+head_hash=$(git log -n1 --format=%H)
+echo "po status for commit ${head_hash}:" > $postatus_file
+echo -e "(if nothing appears below, no errors were detected)\\n" >> $postatus_file
+
+function lintpo() {
```

Comment:
What happens if subsequent commits have errors?

Reply:
The script will keep backtracking until it finds a file which passes linting - the beauty of doing this on a git repo is that since all the files right now pass linting, we know that there'll always be a file we can (eventually) get back to which passes. That may take rather a while in the worst case, but if we notice there's a locale which is failing for a few builds in a row we can always submit a PR to fix the error.